GetMessage/index.ts

```diff
@@ -18,18 +17,22 @@ import createAzureFunctionHandler from "io-functions-express/dist/src/createAzur
 import { cosmosdbInstance } from "../utils/cosmosdb";
 import { GetMessage } from "./handler";
 
+import { getConfigOrThrow } from "../utils/config";
+
 // Setup Express
 const app = express();
 secureExpressApp(app);
 
-const messageContainerName = getRequiredStringEnv("MESSAGE_CONTAINER_NAME");
+const config = getConfigOrThrow();
+
+const messageContainerName = config.MESSAGE_CONTAINER_NAME;
 
 const messageModel = new MessageModel(
   cosmosdbInstance.container(MESSAGE_COLLECTION_NAME),
   messageContainerName
 );

```

Comment:
Can we avoid when possible constant aliasing?
```suggestion
const messageModel = new MessageModel(
  cosmosdbInstance.container(MESSAGE_COLLECTION_NAME),
  config.MESSAGE_CONTAINER_NAME
);
```

Reply:
Ok thanks